cl/phase1/forkchoice/on_block.go

```diff
@@ -68,6 +69,15 @@ func verifyKzgCommitmentsAgainstTransactions(cfg *clparams.BeaconChainConfig, bl
 	return ethutils.ValidateBlobs(block.BlobGasUsed, cfg.MaxBlobGasPerBlock, cfg.MaxBlobsPerBlock, expectedBlobHashes, &transactions)
 }
 
+func collectOnBlockLatencyToUnixTime(ethClock eth_clock.EthereumClock, slot uint64) {
+	currSlot := ethClock.GetCurrentSlot()
+	if slot != currSlot {
+		return
+	}
+	initialSlotTime := ethClock.GetSlotTime(slot)
+	monitor.ObserveBlockImportingLatency(initialSlotTime)
```

Comment:
According to the naming `ObserveBlockImportingLatency`, I guess you need slotTimeOf(curSlotTime -  highestSeenSlotTime)?

Reply:
ok i understand it

Reply:
I prefer to compute latency outside the function, which is clear.